webview/src/plots/components/comparisonTable/cell/ComparisonTableCell.tsx

```diff
@@ -29,12 +35,21 @@ export const ComparisonTableCell: React.FC<{
       onClick={() => zoomPlot(plotImg.url)}
       data-testid="image-plot-button"
     >
-      <img
-        className={styles.image}
-        draggable={false}
-        src={plotImg.url}
-        alt={imgAlt || `Plot of ${path} (${plot.id})`}
-      />
+      {plotImg.url && plotImg.boundingBoxes ? (
+        <ComparisonTableBoundingBoxImg
+          src={plotImg.url}
+          boxCoords={plotImg.boundingBoxes}
```

Comment:
Instead of passing down complex objects (here and further up), can we just pass down an id and go grab the coordinates where they are used instead?

Reply:
Good point on the fact that we're passing down complex objects! I'll look into moving the bounding box data into an object that `ComparisonTableBoundingBoxImg` could grab from.